util/helm/cmd.go

```diff
@@ -63,13 +69,9 @@ func (c Cmd) run(ctx context.Context, args ...string) (string, string, error) {
 			fmt.Sprintf("HELM_CONFIG_HOME=%s/config", c.helmHome))
 	}
 
-	if c.IsHelmOci {
-		cmd.Env = append(cmd.Env, "HELM_EXPERIMENTAL_OCI=1")
```

Comment:
Should we remove `HELM_EXPERIMENTAL_OCI` also from other places in the code that use it?

Reply:
Done!